ddtrace/llmobs/_integrations/bedrock.py

```diff
@@ -55,3 +55,39 @@ def _llmobs_metrics(span: Span, formatted_response: Optional[Dict[str, Any]]) ->
             metrics["completion_tokens"] = completion_tokens
             metrics["total_tokens"] = prompt_tokens + completion_tokens
         return metrics
+
+    @staticmethod
+    def _extract_input_message(prompt):
+        """Extract input messages from the stored prompt.
+        Anthropic allows for messages and multiple texts in a message, which requires some special casing.
+        """
+        if isinstance(prompt, str):
+            return [{"content": prompt}]
+        if not isinstance(prompt, list):
+            return [{"content": ""}]
+        input_messages = []
+        for p in prompt:
+            content = p.get("content", "")
+            if isinstance(content, list) and isinstance(content[0], dict):
+                for entry in content:
+                    if entry.get("type") == "text":
+                        input_messages.append({"content": entry.get("text"), "role": str(p.get("role"))})
+                    elif entry.get("type") == "image":
+                        # Store a placeholder for potentially enormous binary image data.
+                        input_messages.append({"content": "{IMAGE DETECTED}", "role": str(p.get("role"))})
```

Comment:
should we mention this in the release notes or docs somewhere? 

A bit hesitant about this since an LLM can also just generate `{IMAGE DETECTED}`. If we do set a placeholder, maybe we should consider making the same as how we surface sensitive information scanning `([IMAGE DETECTED])` so the way we display placeholder text is standardized. I can also see an argument for wanting these two placeholders to have separate displays though

Reply:
We can move forward with being consistent with the sensitive information scanning placeholder.
For regular APM users, this placeholder won't matter (since they will still get the truncated image blob tag anyway) so we don't need to add into a release note. For LLMObs customers, we'll just have to add this to our internal/onboarding docs so customers are aware we don't support submitting entire image blobs.